lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/auth/DBOAuthenticationDAOImpl.java

```diff
@@ -167,16 +166,16 @@ public boolean checkUserCredentials(long principalId, String passHash) {
 	@WriteTransaction
 	public boolean revalidateSessionTokenIfNeeded(long principalId) {
 		// Determine the last time the token was re-validate.
-		Long lastValidatedOn = jdbcTemplate.queryForObject(SELECT_AUTHENTICATED_ON_FOR_PRINCIPAL_ID, new SingleColumnRowMapper<Long>(), principalId);
+		Date lastValidatedOn = jdbcTemplate.queryForObject(SELECT_AUTHENTICATED_ON_FOR_PRINCIPAL_ID, new SingleColumnRowMapper<Date>(), principalId);
```

Comment:
This can use the Date.class instead of the single column row mapper: jdbcTemplate.queryForObject(SELECT_AUTHENTICATED_ON_FOR_PRINCIPAL_ID, Date.class, principalId)

Reply:
Gotcha.  There are actually a bunch of places this change can be made in this class.  I'll update the next PR.